charts/linkerd2/values.yaml

```diff
@@ -149,6 +149,13 @@ prometheusRuleConfigMapMounts: []
   # - name: recording-rules
   #   subPath: recording_rules.yml
   #   configMap: linkerd-prometheus-rules
+prometheusPersistence:
+  ### WARNING: prometheusPersistence is experimental and has not been tested/vetted by the Linkerd team.
+  ### As such, please refer to https://linkerd.io/2/tasks/exporting-metrics/ for the recommended approach to metrics data retention.
```

Comment:
@grampelberg are you ok with this wording or should we use the more concise "this is not a guarantee your data will actually persist, if you care about your data, federate" you suggested?